veps/sig-network/ns-admin-nichotplug.md

```diff
@@ -0,0 +1,212 @@
+# VEP #29: NIC Hotplug / Hotunplug For Namespace Admins Following VMIM Hardening
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+Currently, the NIC hotplug / hotunplug feature depends on a user-initiated migration following each NIC hotplug / hotunplug
+operation.
+The migration is required in order to attach / detach secondary interfaces to / from the target virt-launcher pod.
+
+> [!NOTE]
+> KubeVirt also supports in-place hotplug / hotunplug operations (without performing a migration)
+> by integrating with the [dynamic-networks-controller](https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller).
+>
+> This method does not support hot-plugging SR-IOV NICs, as SR-IOV hotplug operation requires a migration in order for the
+> [SR-IOV device plugin](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin) to be invoked when the
+> target pod is created.
+
+PR [kubevirt/kubevirt#13497](https://github.com/kubevirt/kubevirt/pull/13497) removed Namespace administrators'
+permissions to create VirtualMachineInstanceMigration objects - thus breaking the NIC hotplug / hotunplug feature
+for them in v1.5.
+
+## Motivation
+
+In previous KubeVirt versions, a namespace admin could hotplug and unplug NICs.
+We want to ensure that this feature will continue working in v1.5 and future versions.
+
+## Goals
+
+Enable namespace admins to perform NIC hotplug / unplug operations independently - without involving the cluster-admin 
```

Comment:
Please convert it to a bullet.

I think the main goal is to integrate the network hotplug to the existing rollout feature, for consistency.

Reply:
Done.